node/src/components/rpc_server.rs

```diff
@@ -178,8 +178,8 @@ where
         match event {
             Event::RpcRequest(RpcRequest::SubmitDeploy { deploy, responder }) => {
                 let mut effects = effect_builder.announce_deploy_received(deploy).ignore();
-                effects.extend(responder.respond(()).ignore());
-                effects
+                effects.extend(responder.respond(()).ignore()); // send responder, then inside return is_valid (ie not a 200)
+                effects // check account in is_valid
```

Comment:
Should that be a TODO?

Reply:
Thanks, I left these here accidentally. They were merely in-place notes taken during a meeting and I didn't intend to commit them. They don't have to do with this PR.